src/api/Jalasoft.TeamUp.Projects.API/local.settings.json

```diff
@@ -5,6 +5,7 @@
     "FUNCTIONS_WORKER_RUNTIME": "dotnet"
   },
   "Host": {
+    "LocalHttpPort": 7072,
```

Comment:
I think this was for your dev testing, please could you undo this change?

Reply:
This is for testing this api while resumes api is running at the same port on our machines.